cpp/src/link_prediction/similarity_impl.cuh

```diff
@@ -36,13 +36,16 @@
 namespace cugraph {
 namespace detail {
 
+enum Coefficient { JACCARD, SORENSEN, OVERLAP, COSINE };
```

Comment:
`enum class (similarity_)coefficient_t` to be consistent with the rest of the codebase.

Reply:
Changed